src/A2A.AspNetCore/A2AEndpointRouteBuilderExtensions.cs

```diff
@@ -84,19 +85,21 @@ public static IEndpointConventionBuilder MapHttpA2A(this IEndpointRouteBuilder e
         routeGroup.MapPost("/tasks/{id}/send", (string id, [FromBody] MessageSendParams sendParams, int? historyLength, string? metadata) =>
             A2AHttpProcessor.SendTaskMessage(taskManager, logger, id, sendParams, historyLength, metadata));
 
-        // /tasks/{id}/sendSubscribe endpoint
-        routeGroup.MapPost("/tasks/{id}/sendSubscribe", (string id, [FromBody] MessageSendParams sendParams, int? historyLength, string? metadata) =>
+        // /v1/tasks/{id}/sendSubscribe endpoint
+        routeGroup.MapPost("v1/tasks/{id}/subscribe", (string id, [FromBody] MessageSendParams sendParams, int? historyLength, string? metadata) =>
```

Comment:
Subscribe is no longer a method.  The replacement is message/send https://a2aproject.github.io/A2A/latest/specification/#72-messagestream
which is exposed for HTTP as message:stream
https://github.com/a2aproject/A2A/blob/6ff9d5de9e0995f3db2fc15effc550f353b08b83/specification/grpc/a2a.proto#L44